packages/aws-cdk/lib/api/cxapp/exec.ts

```diff
@@ -73,6 +74,10 @@ export async function execProgram(aws: ISDK, config: Configuration): Promise<cxa
   debug('outdir:', outdir);
   env[cxapi.OUTDIR_ENV] = outdir;
 
+  // Send version information
+  env[cxapi.CLI_ASM_VERSION_ENV] = cxapi.CLOUD_ASSEMBLY_VERSION;
```

Comment:
Curious why through environment variables and not through context... not that I have a strong opinion...